server/game/cards/07-WotW/ShinjoTrailblazer.js

```diff
@@ -0,0 +1,19 @@
+const DrawCard = require('../../drawcard.js');
+
+class ShinjoTrailblazer extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Gain +2/+2',
+            when: {
+                onCardRevealed: (event, context) => event.card.isProvince && event.card.controller === context.source.controller.opponent && this.game.isDuringConflict()
+            },
+            gameAction: ability.actions.cardLastingEffect({ effect: ability.effects.modifyBothSkills(2) }),
+            effect: 'to give {0} +2{1}, +2{2}',
+            effectArgs: () => ['military', 'political']
```

Comment:
You don't need to do this any more - the new chat log looks for military and political within string text

Reply:
So just to clarify we would just have

```
effect: 'to give {0} +2military, +2political',
```
with no spaces?

Reply:
Huh, you're right I hadn't thought of that.  It will  only replace them as separate words (i.e. with spaces on both sides...)